src/screens/AboutApp/AboutApp.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { ScrollView, View, Text, Image } from "react-native";
+import { translate } from "../../i18n";
+import { Button } from "../../components";
+
+import styles from "./styles";
+
+const logo = require("../../assets/logotype/Logo.png");
```

Comment:
U renamed 'logo.png' to 'Logo.png'. But other screens still require 'logo.png' and therefore we have packager issue. Pls, fix imports in all files.

<img width="345" alt="Screen Shot 2020-01-08 at 10 26 53 PM" src="https://user-images.githubusercontent.com/58725560/72008981-ff07ac00-3265-11ea-9540-061c70749f6e.png">

Reply:
fixed